### PR TITLE
Feature/linter format GitHub actions

### PR DESCRIPTION
**Pull request by htuzel**
## What is changed

- Created linter and format actions

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '14'
```
Comment (sbellone) [inline .github/workflows/lint.yml:16]: Node 14 has been deprecated for while now.

File: .github/workflows/format.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+    
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '14'
```
Comment (sbellone) [inline .github/workflows/format.yml:18]: Node 14 has been deprecated for while now.

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: Lint Code
+
+on:
+  pull_request:
+    branches: [ develop ]
```
Comment (sbellone) [inline .github/workflows/lint.yml:5]: You should make it run automatically on any PR to be able to fix the linting before merging.

File: .github/workflows/format.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        ref: ${{ github.head_ref }}
+        fetch-depth: 0
+    
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+    
+    - name: Install dependencies
+      run: npm ci --legacy-peer-deps
```
Comment (sbellone) [inline .github/workflows/format.yml:23]: Can you explain why you need this flag? I see that its usage is not recommended: https://docs.npmjs.com/cli/v10/using-npm/config#legacy-peer-deps

Comment (htuzel) [reply .github/workflows/format.yml]: Good point. Thank you for raising it.  `@algolia/ui-components-horizontal-slider-react@1.2.2` was not compatible with React 18, and I reported it to the front-end team. The front-end team said it is safe to use `--legacy-peer-deps` for now, but it also seems they released 1.2.3, and it is working fine without the flag.

**Review by sbellone — APPROVED** [review]

**Pull request merged**